partner_multi_company/models/res_partner.py

```diff
@@ -12,6 +12,9 @@ class ResPartner(models.Model):
     _name = 'res.partner'
     _description = 'Partners (Multi-Company)'
 
+    display_name = fields.Char(compute='_compute_display_name', store=True,
```

Comment:
Why do you need to re-add this field?

Reply:
This change is because in Sale/Customers no search by name. With this change if yes work.